specification/caas-published-cohort-definitions.yaml

```diff
@@ -27,6 +27,188 @@ info:
+                "variables": ""
+              }
+            },
+            "url": {
+              "raw": "https://internal-dev.api.service.nhs.uk/caas-published-cohort-definitions/api",
+              "protocol": "https",
+              "host": [
+                "internal-dev",
+                "api",
+                "service",
+                "nhs",
+                "uk"
+              ],
+              "path": [
+                "caas-published-cohort-definitions",
+                "api"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "PublishedCohortLibraryGetAll - SANDBOX",
+          "request": {
+            "method": "POST",
+            "header": [],
+            "body": {
+              "mode": "graphql",
+              "graphql": {
+                "query": "query PublishedCohortLibraryGetAll {\n    PublishedCohortLibraryGetAll {\n        authors\n        clinicalAtRiskGroupsText\n        id\n        commissioner\n        demographicsText\n        description\n        disclaimerText\n        fixedDateReference\n        lastUpdated\n        name\n        purpose\n        shortName\n        summary\n        urlSlug\n    }\n}",
+                "variables": ""
+              }
+            },
+            "url": {
+              "raw": "https://internal-dev-sandbox.api.service.nhs.uk/caas-published-cohort-definitions/api",
+              "protocol": "https",
+              "host": [
+                "internal-dev-sandbox",
+                "api",
+                "service",
+                "nhs",
+                "uk"
+              ],
+              "path": [
+                "caas-published-cohort-definitions",
+                "api"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "PublishedCohortLibraryGetAll",
+          "request": {
+            "auth": {
+              "type": "apikey",
+              "apikey": [
+                {
+                  "key": "value",
+                  "value": "lVlQRHlM4M111q8fnmLBe201HAWMNQJH16SU8Q4C",
+                  "type": "string"
+                },
+                {
+                  "key": "key",
+                  "value": "x-api-key",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "POST",
+            "header": [],
+            "body": {
+              "mode": "graphql",
+              "graphql": {
+                "query": "query PublishedCohortLibraryGetAll {\n    PublishedCohortLibraryGetAll {\n        authors\n        clinicalAtRiskGroupsText\n        id\n        commissioner\n        demographicsText\n        description\n        disclaimerText\n        fixedDateReference\n        name\n        purpose\n        shortName\n        summary\n        urlSlug\n    }\n}",
```

Comment:
These queries don't show any nested elements, it would seem a good idea to show some nested elements so users get a better feel how to create queries.

Reply:
✅ added nested query